deploy/roles/revolution-r/tasks/main.yml

```diff
@@ -31,8 +31,7 @@
 
 - name: Ensure that R is executable
   file:
-    #dest: /usr/lib64/R/etc/{{ item }}
-    dest: /usr/lib64/RRO-3.2.0/R-3.2.0/lib64/R/etc/{{ item }}
+    dest: {{ revolution_r_path }}{{ item }}
```

Comment:
@olivermel Sorry to be nitpicky (and I should have mentioned this originally), but could we make the variable default be "/usr/lib64/RRO-3.2.0/R-3.2.0" and have the "/lib64/R/etc/{{ item }}" part be hard coded?  That should always be the same.